projects/js-packages/publicize-components/src/components/connection-management/connection-status.tsx

```diff
@@ -0,0 +1,42 @@
+import { __ } from '@wordpress/i18n';
+import { Connection } from '../../social-store/types';
+import { Disconnect } from './disconnect';
+
+export type ConnectionStatusProps = {
+	connection: Connection;
+	onReconnect?: VoidFunction;
+};
+
+/**
+ * Connection status component
+ *
+ * @param {ConnectionStatusProps} props - component props
+ *
+ * @returns {import('react').ReactNode} - React element
+ */
+export function ConnectionStatus( { connection, onReconnect }: ConnectionStatusProps ) {
+	if ( connection.status === 'ok' ) {
```

Comment:
This is never happens to be the case for me and all the connections show as broken

Reply:
Yeah, the reason for that is the connection data is incomplete as of now unless this is done https://github.com/Automattic/jetpack-reach/issues/310